.github/workflows/publish.yml

```
@@ -67,14 +67,48 @@ jobs:
     - build
     environment:
       name: pypi
-      url: https://test.pypi.org/p/test-pyqir
+      url: https://pypi.org/p/iqm-pyqir
     permissions:
       id-token: write
     steps:
       - name: Download distribution packages
         uses: actions/download-artifact@v4
+        with:
+          path: dist/
+      - name: Flatten artifact structure
```

To be just a little bit more precise:

```suggestion
      - name: Flatten artifact folder structure
```